content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -193,6 +193,100 @@ it does the following when a Pod is created:
 1. If the spec of the incoming Pod doesn't already contain any `imagePullSecrets`, then the
    admission controller adds `imagePullSecrets`, copying them from the `ServiceAccount`.
 
+### Legacy ServiceAccount token cleaner
+
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+
+Starting with Kubernetes version 1.28, a new feature called
+`LegacyServiceAccountTokenTracking` is enabled by default. This feature creates
+a ConfigMap called `kube-apiserver-legacy-service-account-token-tracking` in the
+`kube-system` namespace. This ConfigMap stores the current date as the tracking
+date. With this feature enabled, each time a legacy service account token is
+used, the Secret label `kubernetes.io/legacy-token-last-used` is updated with
+the latest usage date.
+
+Starting with Kubernetes version 1.29, the legacy ServiceAccount token cleaner
+is enabled by default. It runs as part of the `kube-controller-manager` and
+checks every 24 hours to see if any legacy service account token has not been
+used in a *specified amount of time*. If so, the cleaner marks those tokens as
+invalid.
+
+The cleaner works by first checking the ConfigMap created by the
+`LegacyServiceAccountTokenTracking` feature. If the current time is a
+*specified amount of time* after the date in the ConfigMap, the cleaner then
+loops through the list of Secrets in the cluster and evaluates each Secret that
+has the type `kubernetes.io/service-account-token`.
+
+If a Secret meets all of the following conditions, the cleaner marks it as
+invalid:
+
+- The Secret is auto-generated, meaning that it is bi-directionally referenced
+  by a ServiceAccount.
+- The Secret is not currently mounted by any pods.
+- The Secret has not been used in a *specified amount of time* since it was
+  created or since it was last used.
+
+The cleaner marks an invalid Secret by adding a label called
+`kubernetes.io/legacy-token-invalid-since` to the Secret, with the current date
+as the value. If an invalid Secret is not used in a *specified amount of time*,
+the cleaner will delete it.
+
+{{< note >}}
+All the *specified amount of time* above is default to be one year. Users can
+configure this value through the `--legacy-service-account-token-clean-up-period`
+flag.
```

Comment:
on the kube-controller-manager component, I suppose?

Reply:
```suggestion
command line argument.
```
Outside of the Go ecosystem, “flag” refers to a boolean value (and this isn't a boolean).

Reply:
Revised.